README.md

```diff
@@ -87,27 +66,43 @@ risks introduced are outweighed by the functionalities they support.
+### Authentication with Fine-grained PAT (optional)
+Scorecard can run successfully with the workflow's default `GITHUB_TOKEN`.
+However, the `Branch-Protection` and (experimental) `Webhooks` checks require additional data that isn't accessible with that token.
 
-1. [Create a Personal Access Token](https://github.com/settings/tokens/new?scopes=public_repo,read:org,read:repo_hook,read:discussion) with the following read permissions:
-    - Note: `Token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
-    - Expiration: `No expiration`
-    - Scopes:
-        * `repo > public_repo`                  Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings. **Note**: for private repositories, you need scope `repo`.
-        * `admin:org > read:org`                Optional: not used in current implementation.
-        * `admin:repo_hook > read:repo_hook`    Optional: needed for the experimental [Webhook](https://github.com/ossf/scorecard/blob/main/docs/checks.md#webhooks) check.
-        * `write:discussion > read:discussion`  Optional: not used in current implementation.
+We therefore suggest you create a fine-grained Personal Access Token (PAT) that Scorecard may use for authentication.
 
-![image](/images/tokenscopes.png)
+1. [Create a fine-grained Personal Access Token](https://github.com/settings/personal-access-tokens/new) with the following settings:
+    - Token name: `OpenSSF Scorecard Action - $USER_NAME/$REPO_NAME>`
+      (Note: replace `$USER_NAME/$REPO_NAME` with the names of your organization and repository so you can keep track of your tokens.)
+    - Expiration: Set `Custom` and then set the date to exactly a year in the future (the maximum allowed)
+    - Repository Access: `Only select repositories` and select the desired repository. 
+      Alternatively, set `All repositories` if you wish to use the same token for all your repositories.
+    - Repository Permissions:
+        * `Administration: Read-only`: Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings.
+        * `Metadata: Read-only` will be automatically set when you set `Administration`
+        * `Webhooks: Read-only`: (Optional) required for the experimental [Webhook](https://github.com/ossf/scorecard/blob/main/docs/checks.md#webhooks) check.
+
+    **Disclaimer:** `Administration: read-only` lets the token fetch a lot of information about the project's settings
+    (for a full list, see the queries marked `(read)` in [GitHub's documentation](https://docs.github.com/en/rest/overview/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28#administration)).
+    While these settings are usually private, they aren't dangerous and would not endanger the project if they were ever leaked.
```

Comment:
Maybe add a sentence that Scorecard only fetches endpoints relevant to our checks?

Reply:
@laurentsimon anything else worth highlighting?

Reply:
> Maybe add a sentence that Scorecard only fetches endpoints relevant to our checks?

Done, PTAL.